### PR TITLE
Use yarn to replace bower

### PR DESCRIPTION
**Pull request by victoria34**
The jitpack issue has been fixed.

Fix https://github.com/oncokb/oncokb/issues/1405

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -315,11 +299,11 @@ module.exports = function(grunt) {
...
-        cdnify: {
-            dist: {
-                html: ['<%= oncokb.dist %>/*.html']
-            }
-        },
+        // cdnify: {
+        //     dist: {
+        //         html: ['<%= oncokb.dist %>/*.html']
+        //     }
+        // },
```
Comment (zhx828) [inline Gruntfile.js:306]: Let's remove them

File: app/styles/main.css [hunk 2]
```diff
@@ -8069,7 +8073,7 @@ h5 {
 
 .input-group-addon.primary {
   color: white;
-  background-color: #3276b1;
```
Comment (zhx828) [inline app/styles/main.css:8072]: why is the change?

Comment (victoria34) [reply app/styles/main.css:8072]: I don't know what happened. It looks like the main.scss recompiled after I run grunt build.

File: Gruntfile.js (hunk 1 above)
Comment (victoria34) [reply Gruntfile.js]: Removed

**Comment by zhx828** [comment]
@victoria34 let do a github release before merging.

**Comment by victoria34** [comment]
> @victoria34 let's do a github release before merging.

Sure.

**Pull request merged**